docs/UserGuide.md

```diff
@@ -184,6 +184,10 @@ Examples:
 Remark:
 * We allow the adding of duplicate companies with different start dates and end dates.
 * This is fully intended and not a bug as this can allow users to keep track of different application windows of the same role in the same company.
+* Check out our [FAQ](#faq) to understand how we distinguish duplicate entries.
+* Start date can be added without the end date, and vice versa. However, when both dates are not entered, the record will
```

Comment:
I believe in the case of our app, it is "when either dates are not entered" rather than "when both dates are not entered"

Reply:
```suggestion
* Start date can be added without the end date, and vice versa. However, when either dates are not entered, the record will
```